orchestration/dagster_orchestration/hca_manage/tests/test_dataset_manager.py

```diff
@@ -51,7 +51,7 @@ def test_create_dataset_with_policy(self):
         )
 
         self.manager.data_repo_client.create_dataset.assert_called_once()
-        self.assertEqual(self.manager.data_repo_client.add_dataset_policy_member.call_count, 2)
+        self.assertEqual(self.manager.data_repo_client.add_dataset_policy_member.call_count, 3)
```

Comment:
We can make this assertion a bit stronger and be sure that `monster@firecloud.org` was indeed added:
```suggestion
        calls = [
            call('fake_dataset_id', policy_name='steward', policy_member={'email':'monster@firecloud.org'}),
            call('fake_dataset_id', policy_name='steward', policy_member={'email':'abc@example.com'}),
            call('fake_dataset_id', policy_name='steward', policy_member={'email':'def@example.com'}),
        ]
        self.manager.data_repo_client.add_dataset_policy_member.assert_has_calls(calls, any_order=True)
```

Reply:
thank you! didn't realize we could check for the actual calls themselves using the mock